javascript/javascript3/week3/script.js

```diff
@@ -0,0 +1,240 @@
+class Circle{
+    constructor(x,y,r,startAngle,endAngle,fillColor, ctx){
+        this.x = x;
+        this.y = y;
+        this.r = r;
+        this.startAngle = startAngle;
+        this.endAngle = endAngle;
+        this.fillColor = fillColor;
+        this.ctx = ctx;
```

Comment:
I like that you pass in the canvas, but maybe it would be better to pass it into the draw method?
And ctx is a very vague name, just calling it `target` or `canvas` would probably be better.